packages/components/src/components/page-error/page-error.scss

```diff
@@ -82,5 +82,9 @@
                 flex-direction: column;
             }
         }
+
+        &--none {
+            display: none;
+        }
```

Comment:
@farabi-deriv maybe can we implement absence of button without styles involved?

Reply:
@maryia-matskevich-deriv done ✅